XamarinCommunityToolkit/Views/RangeSlider.shared.cs

```diff
@@ -0,0 +1,615 @@
+﻿using System.Collections.Generic;
+using Xamarin.Forms;
+using Xamarin.Forms.Internals;
+using static System.Math;
+using static Xamarin.Forms.AbsoluteLayout;
+using System.Runtime.CompilerServices;
+
+namespace XamarinCommunityToolkit.Views
```

Comment:
After this is merged and #154 we need to move this to `Microsoft.Toolkit.Xamarin.Forms.UI.Controls`